luxonis_train/nodes/heads/discsubnet_head/discsubnet_head.py

```diff
@@ -109,3 +109,11 @@ def wrap(
                 "reconstructed": [recon],
                 "segmentation": [seg_out],
             }
+
+    def get_custom_head_config(self) -> dict:
+        """Returns custom head configuration.
+
+        @rtype: dict
+        @return: Custom head configuration.
+        """
+        return {}
```

Comment:
AFAIK we should be able to use `SegmentationParser` here as well. CC: @JSabadin  on the correct parameters so this model can work with the dai-nodes parser

Reply:
The default parameters are the correct ones.